src/main.scss

```diff
@@ -9,8 +9,11 @@
 }
 
 body {
-  background-image: url("static/images/Background_cinema.jpg");
+  background-image: url("../images/Background_cinema.jpg");
```

Comment:
Do we think that this is the solution to the problem with the background not showing? Fine in that case. But I think that there must be some other problems too; the text in salon and about us is not displayed correctly either.